server/api/cart.js

```diff
@@ -30,4 +30,63 @@ router.get('/', async (req, res, next) => {
   }
 })
 
-//for the view past orders get, use findall and no status validator
+router.post('/:id', async (req, res, next) => {
+  const id = req.params.id
+
+  //find record matching id
+  const foundRecord = await Record.findOne({
+    where: {
+      id: id
+    }
+  })
+
+  //create new row in Record
+  //!!THIS IS WHERE WE WOULD AFFECT QUANTITY - DOES THE RECORD ALREADY EXIST IN CART? IF SO, SIMPLY CHANGE QUANTITY, DO NOT ADD WHOLE NEW RECORDORDER
+  const newRecordOrder = await RecordOrder.create({
+    quantity: 1,
+    soldPrice: foundRecord.price
+  })
+  //assign recordorder to record
+  newRecordOrder.belongsTo(foundRecord)
```

Comment:
Only add associations on the actual Models, for specific objects: add foreign keys